### PR TITLE
New motd

### PR DESCRIPTION
**Pull request by moul**
Before

```console
$ scw exec -w pensive-mcclintock
Linux pensive-mcclintock 3.2.34-30 #17 SMP Mon Apr 13 15:53:45 UTC 2015 armv7l

Welcome on Debian Wheezy on Scaleway' C1.

 * Kernel:           GNU/Linux 3.2.34-30 armv7l - Marvell (Proprietary)
                     - This kernel has the best performances on this hardware
                     - For mainline kernel with latest features and plenty of modules, use a 3.17 kernel instead
 * Distribution:     Debian Wheezy (2015-09-07) on Debian GNU/Linux 7.9 (wheezy)
 * Internal ip:      10.1.38.7
 * External ip:      212.47.230.153
 * Disk /dev/nbd0:   pensive-mcclintock-scw-distrib-debian-wheezy-2015-09-07_15:07 (l_ssd 50G)
 * Uptime:           12:44:39 up 2 min,  0 users,  load average: 0.19, 0.14, 0.06

Links
 * Documentation:    https://scaleway.com/docs
 * Community:        https://community.scaleway.com
 * Image source:     https://github.com/scaleway/image-debian
root@pensive-mcclintock:~#
```

After

```console
$ scw exec -w pensive-mcclintock
                _
  ___  ___ __ _| | _____      ____ _ _   _
 / __|/ __/ _` | |/ _ \ \ /\ / / _` | | | |
 \__ \ (_| (_| | |  __/\ V  V / (_| | |_| |
 |___/\___\__,_|_|\___| \_/\_/ \__,_|\__, |
                                     |___/

Welcome on Ubuntu Trusty (14.04 LTS) (GNU/Linux 3.2.34-30 armv7l - Marvell)

System information as of: Mon Sep 14 12:45:15 UTC 2015

System load:    0.81            Int IP Address: 10.1.20.115
Memory usage:   4.3%            Pub IP Address: 212.47.229.101
Usage on /:     2%              Swap usage:     0.0%
Local Users:    1               Processes:      80
Image build:    2015-09-04      System uptime:  1:48 hours
Disk nbd0:      l_ssd 50G

Documentation:  https://scaleway.com/docs
Community:      https://community.scaleway.com
Image source:   https://github.com/scaleway/image-ubuntu
root@pensive-mcclintock:~#
```

### CODE REVIEW

**Comment by bchatelard** [comment]
nice :+1:

File: skeleton-feature-motd/etc/update-motd.d/50-scw [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  oc-metadata --cached "VOLUMES_${device}_${key}"
+}
+
+
+KERNEL_VERSION=$(uname -r)
+if [[ $KERNEL_VERSION =~ ^3\.2\.[35][24].* ]]; then
+    KERNEL_TITLE="- Marvell"
+fi
+
+[ -f /etc/motd.head ] && cat /etc/motd.head || true
```
Comment (QuentinPerez) [inline skeleton-feature-motd/etc/update-motd.d/50-scw]: You shouldn't consider `/etc/motd.head` as a script ?

File: skeleton-feature-motd/etc/update-motd.d/50-scw [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+  SIZE=$(( $(volume_metadata $i SIZE) / 1000 / 1000 / 1000 ))G
+  printf "Disk nbd%s:\t%s\n" "$i" "$(volume_metadata $i VOLUME_TYPE) ${SIZE}"
+done
+printf "\n"
+printf "Documentation:\t%s\n" "$IMAGE_DOC_URL"
+printf "Community:\t%s\n" "$IMAGE_HELP_URL"
+printf "Image source:\t%s\n" "$IMAGE_SOURCE_URL"
+printf "\n"
+[ -f /etc/motd.tail ] && cat /etc/motd.tail || true
```
Comment (QuentinPerez) [inline skeleton-feature-motd/etc/update-motd.d/50-scw]: Same as above

File: skeleton-feature-motd/etc/update-motd.d/50-scw (hunk 1 above)
Comment (moul) [inline skeleton-feature-motd/etc/update-motd.d/50-scw]: Nop, `motd.{tail,head}` are standard distrib files, /etc/update.motd.d/* are dynamic scripts

File: skeleton-feature-motd/etc/update-motd.d/50-scw (hunk 2 above)
Comment (moul) [inline skeleton-feature-motd/etc/update-motd.d/50-scw]: Ditto

File: skeleton-feature-motd/etc/update-motd.d/50-scw (hunk 1 above)
Comment (QuentinPerez) [inline skeleton-feature-motd/etc/update-motd.d/50-scw]: oki 😊

**Comment by QuentinPerez** [comment]
LGTM

**Comment by EvaSDK** [comment]
Isn't this specific to debian based distributions ? https://wiki.debian.org/motd

**Comment by edouardb** [comment]
LGTM

File: skeleton-feature-motd/etc/motd.head [hunk 3]
```diff
@@ -0,0 +1,6 @@
+               _
+ ___  ___ __ _| | _____      ____ _ _   _
+/ __|/ __/ _` | |/ _ \ \ /\ / / _` | | | |
+\__ \ (_| (_| | |  __/\ V  V / (_| | |_| |
+|___/\___\__,_|_|\___| \_/\_/ \__,_|\__, |
+                                    |___/
```
Comment (kdeldycke) [inline skeleton-feature-motd/etc/motd.head]: :heart: ASCII-art ! ;)

Comment (moul) [inline skeleton-feature-motd/etc/motd.head]:
```       
                .x~~"*Weu.   
               d8Nu.  9888c  
               88888  98888  
         .o@8  "***"  9888%  
     .o@8888%       ..@8*"   
 .o@8888%"       ````"8Weu   
8888R"          ..    ?8888L 
"*8888bu.     :@88N   '8888N 
   ^"*8888bu. *8888~  '8888F 
       ^%*888 '*8"`   9888%  
           ^%   `~===*%"`    
```

**Pull request merged**